lib/string_foundation/symbol.rb

```diff
@@ -0,0 +1,13 @@
+# ==============================================================================
+# LIB - STRING FOUNDATION - SYMBOL
+# ==============================================================================
+# frozen_string_literal: true
+class String
+
+  def is_sym?(symbol)
+    raise ArgumentError.new("argument must be Symbol") unless symbol.instance_of?(Symbol)
+
+    self == symbol.to_s
```

Comment:
Enclose a returning conditional value in parentheses because of be more readable. Read more [this](https://github.com/brushdown/string_foundation.rb/pull/32#discussion_r145046187).

```rb
(self == symbol.to_s)
```

Reply:
Sorry, I forgot...